src/common/BaseAnnotator.ts

```diff
@@ -98,7 +98,7 @@ export default class BaseAnnotator extends EventEmitter {
         this.store.dispatch(toggleAnnotationModeAction(mode));
     }
 
-    toggleAnnotationVisibility = (): void => {
-        this.store.dispatch(toggleAnnotationVisibilityAction());
+    setAnnotationVisibility = (visibility: boolean): void => {
```

Comment:
In some places, we're using `Annotation` and in some places we're not. Can we standardize on `setVisibility`? The calling pattern would look like `this.annotator.setVisibility(false)`.

Reply:
> In some places, we're using `Annotation` and in some places we're not. Can we standardize on `setVisibility`? The calling pattern would look like `this.annotator.setVisibility(false)`.

Sounds good